### ISSUE TITLE
Support focus follows mouse *without* raise

### ISSUE THREAD

**Issue by pmonks** [synappser/AutoRaise#3]
**Is your feature request related to a problem? Please describe.**
Many users on Linux (and Windows) prefer FFM *without* the focused window being raised.

**Describe the solution you'd like**
It would be great if AutoRaise supported this.

**Describe alternatives you've considered**
No other macOS tools that I'm aware of support FFM *without* raise, so this may be a limitation of the OS.

**Additional context**
n/a

**Comment by synappser** [comment]
Hi @pmonks, I'm going to implement this behaviour as an optional one and even try and combine it with a real-time interactive modifier key.

I'm however still intrigued about the use case for such a behaviour and would love to see you elaborate a little further on when it might be needed. Thank you.

**Comment by pmonks** [comment]
There's no use case per se - it's just a (strong) preference. In several decades of using both, I've never enabled auto-raise on either Linux or Windows.

**Comment by alotia** [comment]
> Hi @pmonks, I'm going to implement this behaviour as an optional one and even try and combine it with a real-time interactive modifier key.
> 
> I'm however still intrigued about the use case for such a behaviour and would love to see you elaborate a little further on when it might be needed. Thank you.

I would love to see this feature, too. One use case I can provide is: I am reading something from, say, Stackoverflow and trying to type the equivalent thing into my local IDE, which is usually most of the width of the screen. I would like to type in the local editor, and view the webpage as well, but the moment the IDE is focussed, it hides the browser.  (Or, I have to tile the windows when doing this.)

This feature was available on some of the old unix desktops (Sun) and I found it very useful.

**Comment by synappser** [comment]
@alotia, your use case makes sense. I'm pushing this feature to the top of my todo list. Thank you and happy new year. :tada:

**Comment by alotia** [comment]
@synappser Thank you and I hope this can be done. You can scroll a windows without it getting the focus, but you cannot type into it. I recall reading many years ago that something like this was "impossible" on the Mac given how the window-system works. I really don't know anything about MacOS development, and things may have changed since then.
Thanks for the quick reply and a happy new year to you, too! Cheers!

**Comment by pmonks** [comment]
@alotia yeah I've read similar things. Hopefully @synappser is able to figure it out though - it's been a constant irritant since I switched to macOS a decade and a half ago.

**Comment by synappser** [comment]
Implemented in v0.6.7b.
